docs/source/commands/osnadminchannel.md

```diff
@@ -0,0 +1,207 @@
+# osnadmin channel
+
+The `osnadmin channel` command allows administrators to perform channel-related
+operations on an orderer, such as joining a channel, listing the channels an
+orderer has joined, and removing a channel.
+
+## Syntax
+
+The `osnadmin channel` command has the following subcommands:
+
+  * join 
+  * list
+  * remove
+
+## osnadmin channel
+```
+usage: osnadmin channel <command> [<args> ...]
+
+Channel actions
+
+Flags:
+      --help                     Show context-sensitive help (also try
+                                 --help-long and --help-man).
+  -o, --orderer-address=ORDERER-ADDRESS  
+                                 Endpoint of the OSN
```

Comment:
I am not sure what ORDERER-ADDRESS is meant to refer to. Should this be --orderer-address=ORDERER-ADMIN-LISTENADDRESS?
and instead of Endpoint of the OSN, it should Admin endpoint of the OSN?

Reply:
I can clarify that it's the Admin endpoint of the OSN but the ORDERER-ADDRESS part is just how the CLI library we're using prints that info (I can't change it unless we also change the name of the flag).

Reply:
Updated